griptape/memory/structure/run.py

```diff
@@ -1,13 +1,19 @@
+from __future__ import annotations
+
 import uuid
+from typing import TYPE_CHECKING, Optional
 
 from attrs import Factory, define, field
 
-from griptape.artifacts.base_artifact import BaseArtifact
 from griptape.mixins import SerializableMixin
 
+if TYPE_CHECKING:
+    from griptape.artifacts import BaseArtifact
+
 
-@define
+@define(kw_only=True)
 class Run(SerializableMixin):
-    id: str = field(default=Factory(lambda: uuid.uuid4().hex), kw_only=True, metadata={"serializable": True})
-    input: BaseArtifact = field(kw_only=True, metadata={"serializable": True})
-    output: BaseArtifact = field(kw_only=True, metadata={"serializable": True})
+    id: str = field(default=Factory(lambda: uuid.uuid4().hex), metadata={"serializable": True})
+    meta: Optional[dict] = field(default=None, metadata={"serializable": True})
```

Comment:
I think this should default to `meta: dict[str, Any]` instead of `Optional`. More consistent with [other meta](https://github.com/griptape-ai/griptape/blob/489453eddb0bfeb026a754278b5d3cde546427bb/griptape/artifacts/base_artifact.py#L20) implementations, and then we don't need `merge_dict_opt`.

Reply:
must have been looking at `BaseVectorStoreDriver.Entry`.